projects/templates/projects/update_project.html

```diff
@@ -0,0 +1,31 @@
+{% extends "codeeve/index.html" %}
+
+{% block header %}
+{% endblock %}
+
+{% block content %}
+    <div class="content-section-a">
+
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-5 col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <form action="{% url 'project_details' pk=project.pk %}" method="post" enctype="multipart/form-data">
```

Comment:
Yes is needed, because of the ImageField, otherwise it doesn't upload an image.
https://docs.djangoproject.com/en/1.9/topics/http/file-uploads/